### PR TITLE
Issue Template

### PR DESCRIPTION
**Pull request by HaHaWTH**
This pull request added issue templates to Leaf.

Current added templates:
- Bug report
- Feature request

~~Submit a blank issue has been disabled xixi~~

### CODE REVIEW

**Comment by F3F5** [comment]
return blank

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    attributes:
+      label: Leaf Version
+      description: What version of Leaf are you running? (`/version`)
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Error log (if applicable)
+      description: If you are reporting a console error, upload any relevant log excerpts to either https://mclo.gs/ or https://gist.github.com, save and the paste the link in this box.
```
Comment (PadowYT2) [inline .github/ISSUE_TEMPLATE/bug_report.yml]:
```suggestion
      description: If you are reporting a console error, upload any relevant log excerpts to either https://mclo.gs or https://gist.github.com, save and the paste the link in this box.
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+  - type: input
+    attributes:
+      label: Configuration
+      description: Link of your Leaf & Gale config files, upload any relevant log excerpts to either https://paste.gg/ or https://gist.github.com, save and the paste the link in this box.
```
Comment (PadowYT2) [inline .github/ISSUE_TEMPLATE/bug_report.yml]:
```suggestion
      description: Link of your Leaf & Gale config files, upload any relevant log excerpts to either https://paste.gg or https://gist.github.com, save and the paste the link in this box.
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    validations:
+      required: true
```
Comment (PadowYT2) [inline .github/ISSUE_TEMPLATE/bug_report.yml]: Why is this required? It could be unrelevant

**Review by PadowYT2 — CHANGES_REQUESTED** [review]
:eyes:

**Comment by Dreeam-qwq** [comment]
🚀🚀🚀
